docs/source/myrtlespeech/data/sampler.rst

```diff
@@ -0,0 +1,12 @@
+============
+ sampler
+============
+
+.. autoclass:: myrtlespeech.data.sampler.SequentialRandomSampler
+    :members:
+    :show-inheritance:
+
+
+.. autoclass:: myrtlespeech.data.sampler.SortaGrad
+    :members:
+    :show-inheritance:
```

Comment:
Can this be auto-generated to reduce chance of forgetting to update it in the future?

```
.. automodule:: myrtlespeech.data.sampler                                        
     :members:                                                                    
     :show-inheritance: 
```

Reply:
I have added this at the beginning of the file